gulpfile.js

```diff
@@ -1460,3 +1462,6 @@ gulp.task('watch', 'Watches for changes in files, re-builds when detected',
     });
 gulp.task('build-experiments', 'Builds experiments.html/js', buildExperiments);
 gulp.task('build-login-done', 'Builds login-done.html/js', buildLoginDone);
+gulp.task('build-validator-light', 'Builds the light version of validator',
```

Comment:
This should go into [build-system/tasks/validator.js](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/validator.js#L36)

Reply:
ah yes. thanks Raghu